dash/dash.py

```diff
@@ -1565,6 +1566,33 @@ def get_asset_url(self, path):
 
         return asset
 
+    def get_relative_path(self, path):
```

Comment:
Naming options:

- `get_relative_path`
- `get_relative_url`
- `get_rel_path`
- `get_rel_url`
- `relative_path`
- `relative_url`
- `rel_path`
- `rel_url`
- `relpath`
- `relurl`

Reply:
- It is technically a `path` and not a full URL although folks might think that "path" is corresponding to the file system path.
- We use `get_` in other places like `get_asset_url`. Do we keep it for consistency? I'd prefer to move `get_asset_url` usage over to this function, so folks just need to learn e.g. `app.relpath('/assets/logo.png')` and `app.relpath('/page-2')`